blog/content/post/2020-12-10-OSFC_2020_part2.md

```diff
@@ -0,0 +1,170 @@
+the x86 part of Zen CPUs. Ron presented oreboot on the AMD Fam17h. He discussed
+the main problem of the Open System Firmware requirement for buildable,
+installable, redistributable firmware, the approaches to it, and the next steps.
+
+* [Linuxboot on HPE platform where do we stand](https://vimeo.com/488143135) by [Jean-Marie Verdun](https://twitter.com/@vejmarie)
+
+ The important status of the project that aims at porting linuxboot on HPE
+platforms. A brief description of a live demo with the approached challenges,
+proof of concept, and next steps. HPE seems to provide LinuxBoot-enabled
+Facebook servers – "private customer." Thanks
+[HPE](https://www.hpe.com/us/en/home.html) for the status.
+
+* [TrenchBoot DRTM features for AMD platforms](https://vimeo.com/488140434) by [Michał Żygowski](https://twitter.com/_miczyg_) and [Piotr Król](https://twitter.com/pietrushnic)
+
+The second talk of our team describing the progress of the
+[TrenchBoot](https://opensource.3mdeb.com/projects/trenchboot.html) project.
+Overall status, added features and 3mdeb support development for the AMD Secure
+Startup. Presenters described the most key changes introduced into the
+project: the DRTM event log and the possibility to boot Xen Hypervisor with
+measured launch.
+
+### Day 3
+
+* [oreboot status report](https://vimeo.com/488139577) by Ryan O'Leary, [Ronald g. Minnich](https://twitter.com/coreboot)
+
+oreboot is a downstream fork of coreboot, with all C removed, and all code
+written in Rust. Since we spoke of oreboot a year ago, a lot has happened, and
+we want to go over the status.
+
+We've got a few interesting facts:
+* oreboot works on booting TockOS (Rust)
+* it can be tried on QEMU
+* works with OpenTITAN chips (FPGA)
+
+[Zephyr as an open platform for Google's embedded controllers](https://vimeo.com/488150463) by [Paul Fagerburg](https://cfp.osfc.io/osfc2020/speaker/ZZDUXC/), [Jack Rosenthal](https://cfp.osfc.io/osfc2020/speaker/H3RYF8/)
+
+Interesting facts about the benefits of moving to a community-driven RTOS
+instead of using our custom kernel, presentation of ChromiumOS Embedded
+Controller and information about google plans in contributing to Zephyr OS.
+
+* [Arm SystemReady and the UEFI firmware ecosystem](https://vimeo.com/488131661) by [Samer El-Haj-Mahmoud](https://cfp.osfc.io/osfc2020/speaker/3DPRVF/), [Dong Wei](https://cfp.osfc.io/osfc2020/speaker/8FY7QG/)
+
+This talk introduces the Arm SystemReady program, the Base Boot Requirements
+(BBR) and the Base Boot Security Requirements (BBSR) firmware specifications.
+The session show-cases enablement efforts for devices under this program, using
+open source firmware projects such as TianoCore and U-Boot.
+
+Just a few keynotes:
+arm ServerReady certificate applied to embedded systems
+* SBSA/SBBR/ACS
+* for embedded BSA/BBR/ACS
+* SBBR/EBBR/LBBR
+* BBSR/EBBR BaseBoot Security and Embedded BBR
+* there is no ISV ecosystem that Arm endorse for certification
```

Comment:
@KBanecka this should be rewritten maybe by @TomaszAIR or @macpijan

Reply:
done by @TomaszAIR